libs/ui/src/lib/fire-calculator/fire-calculator.component.ts

```diff
@@ -50,20 +51,24 @@ export class FireCalculatorComponent
   @Input() hasPermissionToUpdateUserSettings: boolean;
   @Input() locale: string;
   @Input() savingsRate = 0;
+  @Input() targetNetWorth = 0;
 
   @Output() savingsRateChanged = new EventEmitter<number>();
+  @Output() targetNetWorthChanged = new EventEmitter<number>();
 
   @ViewChild('chartCanvas') chartCanvas;
 
   public calculatorForm = this.formBuilder.group({
     annualInterestRate: new FormControl<number>(undefined),
     paymentPerPeriod: new FormControl<number>(undefined),
     principalInvestmentAmount: new FormControl<number>(undefined),
-    time: new FormControl<number>(undefined)
+    time: new FormControl<number>(undefined),
+    retirementNetWorth: new FormControl<number>(undefined)
```

Comment:
Please move up (for an alphabetical order)

Reply:
Done